app/test_main.py

```diff
@@ -2,0 +1,11 @@
+from app.main import can_access_google_page
+
+
+def test_can_acces_google_page_for_acessible() -> None:
+    url = "https://example.com"
+    assert can_access_google_page(url) == "Accessible"
+
+
+def test_can_acces_google_page_for_not_acessible() -> None:
+    url = "https://example.com/nonexistentpage"
+    assert can_access_google_page(url) == "Not accessible"
```

Comment:
Similar to the previous test, this test does not mock the `valid_google_url` and `has_internet_connection` functions. Ensure these are mocked to properly test `can_access_google_page`.